ribs.py

```diff
@@ -0,0 +1,132 @@
+#
+# The offical LiTHe kod code skeleton for PyGame development.
+# 2020 by Edvard Thörnros
+#
+# The aim of this file is to give a collection of usable functions
+# and structures to be used as a jumping off point. Feel free
+# to change and distribute!
+
+import pygame as pg
+# For smaller cheat sheet see:
+#   https://www.lithekod.se/pygame.html
+# For official pygame documentation see:
+#   https://www.pygame.org/docs/
+
+#
+# Utility
+#
+
+running = True
+def is_running():
+    """Is the game currently running?"""
+    return running
+
+
+def quit_game():
+    """Pull the brakes and stop the game!"""
+    global running
```

Comment:
Don't really like encouraging globals, but perhaps there is no better way...

Reply:
It's atleased encapsulated in a function. But how would you do it without global state? I can't think of a "simple" way to do it.